TweetMate/src/main/java/com/ybsystem/tweetmate/resources/ResColor.java

```diff
@@ -0,0 +1,88 @@
+package com.ybsystem.tweetmate.resources;
+
+import android.util.TypedValue;
+
+import com.ybsystem.tweetmate.R;
+import com.ybsystem.tweetmate.application.TweetMateApp;
+import com.ybsystem.tweetmate.databases.PrefWallpaper;
+
+import static com.ybsystem.tweetmate.resources.ResString.*;
+
+public class ResColor {
+
+    public static int COLOR_BACKGROUND;
+    public static int COLOR_TEXT;
+    public static int COLOR_PRIMARY;
+    public static int COLOR_ACCENT;
+    public static int COLOR_HIGHLIGHT;
+
+    public static int COLOR_WEAK;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this "public static COLOR_WEAK" field final [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS1444)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make COLOR_WEAK a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)